Doc update (#166)
Scalr/terraform-provider-scalr: docs/data-sources/scalr_tag.md

# Data Source `scalr_tag` 

> Retrieves information about the tag.

Reviewer: ```suggestion
Retrieves information about a tag.
```

## Arguments

> * `name` - (Required) A name of the tag.

Reviewer: ```suggestion
* `name` - (Required) The name of the tag.
```

> * `account_id` - (Required) An ID of the Sclar account, in the format `acc-<RANDOM STRING>`

Reviewer: ```suggestion
* `account_id` - (Required) The ID of the Scalr account, in the format `acc-<RANDOM STRING>`
```
